Sources/Brave/Frontend/Browser/New Tab Page/Sections/BraveNewsSectionProvider.swift

```diff
@@ -375,9 +386,15 @@ class BraveNewsSectionProvider: NSObject, NTPObservableSectionProvider {
     }
   }
 
-  private func handler(from feedList: @escaping (Int) -> FeedItem, card: FeedCard, indexPath: IndexPath) -> (Int, FeedItemAction) -> Void {
+  private func handler(from feedList: @escaping (Int) -> FeedItem?, card: FeedCard, indexPath: IndexPath, isRatingCardPair: Bool = false) -> (Int, FeedItemAction) -> Void {
```

Comment:
Lets separate this out like we did context menu

Reply:
Separation is done both for handler and context menu fix at https://github.com/brave/brave-ios/pull/7589/commits/ebbe26fa4af9e4be47f82ac8e078025b0f0f5031